vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -455,6 +455,17 @@ public Connection getWatchLaterConnection() {
     }
     // </editor-fold>
 
+    // <editor-fold desc="Channel">
+    @Nullable
+    public Interaction getChannelMembershipInteraction() {
+        if (mMetadata != null && mMetadata.getInteractions() != null && mMetadata.getInteractions().getChannelMembership() != null) {
+            return mMetadata.getInteractions().getChannelMembership();
+        }
```

Comment:
Small nit 1: Github does horizontal scrolling on this line: does it exceed 120 characters?
Small nit 2: You could pull out the InteractionCollection and the Interaction into variables. FWIW, this would be so much cleaner in Kotlin!

Reply:
Good suggestions.